package/yast2-ycp-ui-bindings.spec

```diff
@@ -20,7 +20,7 @@
 %define yui_so		13
 
 Name:           yast2-ycp-ui-bindings
-Version:        4.3.1
+Version:        4.3.2
```

Comment:
I am not sure whether bumping the version is actually needed only because this change in the example.

Reply:
Maybe this example should be provided by yast-ruby-bindings instead of yast-ycp-ui-bindings?

Reply:
It's UI examples, so the UI interpreter is the right place.

But since the UI interpreter is no longer parsing YCP, the name and everything are very much outdated; this is an artifact of the YCP-to-Ruby migration.

But as always, there is a chicken-egg problem when a new widget is introduced: The ruby bindings should really fetch those symbols from the UI rather than having an outdated copy of them in this obscure Ruby source file.